.github/workflows/linting.yml

```diff
@@ -1,40 +1,22 @@
 name: Linting
 on:
+  push:
+    branches:
+      - master
   pull_request:
-    paths:
-      - '**/*.yaml'
-      - '**/*.yml'
```

Comment:
Sorry about that, I should have noticed when I moved the configuration around.

Let me just have another look at the documentation before I approve this PR.

Reply:
Ok, so the intention with these two lines was to not run Yamllint unnecessarily. From what I can see, the Yamllint check usually takes about 20 seconds to run, so it’s no big problem to merge the config as-is.

The alternative I was looking into involves using e.g. [the dorny/paths-filter action](https://github.com/dorny/paths-filter) to look at changed paths _inside_ the `lint-yaml` job. But I’m not sure that it will actually be faster than 20 seconds anyway 😅

Reply:
Huh, 20s seems quite long for what it does